tests/certificate/certificate.permission.test.js

```diff
@@ -0,0 +1,123 @@
+const { setupApp } = require('../helper-functions');
+const { INVALID_PERMISSIONS } = require('../../error-messages/user.messages');
+
+const { loginUser, deleteUser } = require('../user/user.helper');
+
+const {
+  CERTIFICATE_NOT_FOUND,
+  CERTIFICATE_IS_ACTIVE,
+} = require('../../error-messages/certificate.messages');
+
+const {
+  addCertificate,
+  deleteCertificate,
+  generateCertificate,
+  getAllCertificates,
+  getCertificateById,
+  registerUser,
+  updateCertificate,
+} = require('./certificate.helper');
+const { newCertificateInputData, newUser } = require('./certificate.variables');
+
+let operations;
+let certificateId;
+let certificateName;
+let certificateNullOwnerId;
+let certificateNullOwnerEmail;
+let certificateNullOwnerName;
+let userId;
+
+describe('Test behaviour of USER with role=user', () => {
```

Comment:
Add describe wrapper, beforeAll will do setupApp, and afterAll clean the db.

Reply:
Thanks, done